### PR TITLE
Support disabling both systemd and initd when installing from tarball.

### PR DESCRIPTION
**Pull request by yanivmn**
This is required for omnibus installations. by default omnibus uses runit as service management.

### CODE REVIEW

File: recipes/tarball.rb [hunk 1]
```diff
@@ -177,7 +177,7 @@
   only_if { ::File.readlines('/etc/pam.d/su').grep(/# #{pam_limits}/).any? }
 end
 
-if node['cassandra']['use_systemd'] == false
+if node['cassandra']['use_initd'] == true
```
Comment (michaelklishin) [inline recipes/tarball.rb]: I've missed this in the original version, let's make this `node['cassandra']['use_initd']`.

File: recipes/tarball.rb [hunk 2]
```diff
@@ -186,7 +186,7 @@
     mode '0755'
     notifies :restart, 'service[cassandra]', :delayed if node['cassandra']['notify_restart']
   end
-else
+elseif node['cassandra']['use_systemd'] == true
```
Comment (michaelklishin) [inline recipes/tarball.rb]: same as above

File: recipes/tarball.rb (hunk 1 above)
Comment (yanivmn) [inline recipes/tarball.rb]: It is already use_initd, can you explain your comment?

Comment (michaelklishin) [inline recipes/tarball.rb]: An explicit comparison with `true` is excessive and looks weird in Ruby. Any value other than `false` or `nil` evaluates to `true`.

**Comment by yanivmn** [comment]
@michaelklishin - can you accept this PR?

**Comment by michaelklishin** [comment]
Now I can, thank you.

**Pull request merged**